packages/components/src/toolbar/stories/index.tsx

```diff
@@ -82,32 +83,28 @@ Default.args = {
 			</ToolbarGroup>
 			<ToolbarGroup>
 				<ToolbarItem>
-					{
-						// @ts-expect-error TODO: Remove when ToolbarItem/DropdownMenu is typed
-						( toggleProps ) => (
-							<DropdownMenu
-								hasArrowIndicator
```

Comment:
The real change here is the removal of `hasArrowIndicator` which has been obsolete since #19344. The rest of this block is just automatic indentation changes.